### ISSUE TITLE
Feature Request: Support for Redux Debugging

### ISSUE THREAD

**Issue by eljubec** [expo/dev-plugins#20]
Hello!

I'm reaching out to request an enhancement for debugging redux within expo.

Previously, with the JSI engine & the remote debugging capability, it was incredibly efficient debugging the redux store when using tools like react native redux-devtools. 

Currently, with the shift to the hermes engine, debugging the redux store is limited to console logging, which is far from ideal due to the lack of support for more advanced debugging tools. 

Having something similar to [react-query-native-devtools](https://github.com/bgaleotti/react-query-native-devtools) that's compatible with hermes would be incredibly valuable and improve the debugging process for many of us.

Thank you for considering this enhancement.

**Comment by jthoward64** [comment]
It doesn't look like a dev plugin would be that difficult, but in the meantime maybe look at https://github.com/reduxjs/redux-devtools/tree/main/packages/redux-devtools-remote

**Comment by Kudo** [comment]
hi there! creating a devtool plugin should be easy and documented at https://docs.expo.dev/debugging/create-devtools-plugins/. you could also try to create one before we get a bandwidth to create the redux plugin.

**Comment by samnatale3** [comment]
This would be perfect

**Comment by dannyBies** [comment]
I've created a basic version -> https://github.com/expo/dev-plugins/pull/28, it's just a read-only view but it's enough for my use-case.

**Comment by matt-oakes** [comment]
I have worked on a full version of this and released it as [redux-devtools-expo-dev-plugin](https://github.com/matt-oakes/redux-devtools-expo-dev-plugin). It is based on the full [Redux DevTools](https://github.com/reduxjs/redux-devtools/) (from the Chrome extension) so you have all of the same features and UI as that.

I was going to submit it here, but [a note was added to the README](https://github.com/expo/dev-plugins/commit/23b6a01af4b6ff2f936352af608b289a9e48a30c) that they're no longer accepting pull requests, so I have published it as it's own package.

The package is published now. I've tested it on one of my projects, but it would be good to get some feedback on it. Feel free to open PRs and issues on the repo.
